### DISCUSSION TITLE
Firefly calling tables before creating it during installation on Windows

### DISCUSSION THREAD

**Discussion by anastutemind** [firefly-iii/firefly-iii#4832] [Firefly III]
**Bug description**
I am running Firefly III version 5.5.12, and my problem is:

During the first install on Windows using Xampp 3.3.0  with PHP 8.0.6
uncommented extension=intl  and extension=gd
change DBhost=local in .env file
created database 'firefly'
then ran composer update, it then stuck to:

In Connection.php line 685:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'firefly.accounts' doesn't exist (SQL: select `id`, `name
  ` from `accounts`)

In Connection.php line 345:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'firefly.accounts' doesn't exist

Script @php artisan firefly-iii:decrypt-all handling the post-update-cmd event returned with error code 1

If I ignore this and run php artisan migrate, it then generate the require tables in firefly database then if I re run composer update then it completes without errors.

I think it is calling tables and user accounts before it creates them.

**Comment by JC5** [comment]
There is no need to call composer update and it is not in the installation instructions to do so.

If you clone Firefly III from github you must run composer install.

Then run

php artisan migrate:refresh --seed
php artisan firefly-iii:upgrade-database
php artisan passport:install

**Reply by anastutemind** [reply]
Thanks, I got it working now; the only warning during the whole process is when I run 'php artisan passport:install' it says
Encryption keys already exist. Use the --force option to overwrite them. Rest all good. Is there a way to avoid running 'php artisan serve' everytime in command line?

**Reply by JC5** [reply]
Well, the A in xampp stands for Apache. I suggest you use that one 😉

**Reply by anastutemind** [reply]
Got it now, thanks
